tests/test_classifier/test_confusion_matrix.py

```diff
@@ -316,14 +285,17 @@ def test_inverse_mapping(self):
             ]
         )
 
-        cm = ConfusionMatrix(model, ax=ax, classes=classes, label_encoder=le)
+        cm = ConfusionMatrix(model, ax=ax, encoder=le)
```

Comment:
Because of the dual nature of `classes` and `encoder` only one should be specified by the user.